react/Carousel.tsx

```diff
@@ -146,7 +147,13 @@ const Carousel = (props: Props) => {
             style={{ maxHeight: height }}
             sliderTransitionDuration={500}
           >
-            <Banner height={height} {...banner} />
+            {i > 1 ? (
```

Comment:
Is this check correct? Shouldn't it be `0` or `>=` ?

Reply:
@iaronaraujo good catch 👌